apps/api/src/server/repositories/representation.repository.js

```diff
@@ -195,6 +195,112 @@ export const createApplicationRepresentation = async ({
 	});
 };
 
+export const updateApplicationRepresentation = async (
+	{ representationDetails, represented, representedAddress, representative, representativeAddress },
+	caseId,
+	representationId
+) => {
+	//  Validate case rep id is on case id
+	const response = await databaseConnector.representation.findFirst({
+		where: { id: representationId, caseId }
+	});
+
+	if (!response) throw new Error('Representation Id and Case Id do not macth');
```

Comment:
May be more useful log -
Representation Id (repId} does not belong to case Id {caseId}